### PR TITLE
Create table components for simple tables

### PR DESCRIPTION
**Pull request by axbarsan**
Towards the `grommet` migration

This PR adds new table building blocks for simple tables (simple meaning no sorting, no resizeable columns or other fancy stuff). Tables refactored:
- Cluster creation instance type selection
- Cluster creation release version selection
- Organizations list view

The other hardcore tables (currently implemented with `react-bootstrap-table-next`) will be refactored in a separate PR.

I've created files for the `grommet` table components and exported them from there without changes, so it would be easy to make changes to all the component instances at once. I plan to eventually do this for all `grommet` components.

### Preview

<details>
<summary>Simple table</summary>

image

</details>

### CODE REVIEW

File: src/components/Users/InvitesTable.js [hunk 1]
```diff
@@ -37,20 +37,20 @@ const getTableColumnsConfig = () => {
     {
       classes: 'created',
       dataField: 'created',
-      text: 'INVITED',
+      text: 'Invited',
```
Comment (axbarsan) [inline src/components/Users/InvitesTable.js:40]: Not within the scope of this PR, but after removing `_table.sass`, `react-bootstrap-table-next` tables no longer have styled headers.

Not a big deal for now, but until the `smart` table refactor.

<details>
<summary>Before</summary>

image

</details>

<details>
<summary>Now</summary>

image

</details>

File: src/components/Cluster/ClusterDetail/InstanceTypeSelector/InstanceTypeSelector.tsx [hunk 2]
```diff
@@ -65,27 +79,41 @@ const InstanceTypeSelector: FC<IInstanceTypeSelector> = ({
...
           >
-            <i className={`fa fa-caret-${collapsed ? 'right' : 'bottom'}`} />
-            Available {plural}
+            <i
+              className={`fa fa-caret-${collapsed ? 'right' : 'bottom'}`}
+              aria-hidden='true'
+              aria-label='Toggle'
+              role='presentation'
+            />
+            <span id='available-machines-label'>Available {plural}</span>
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/InstanceTypeSelector/InstanceTypeSelector.tsx:97]: Cheeky little improvement to make keyboard navigation easier. This was already in the release version selector.

File: src/components/Users/InvitesTable.js (hunk 1 above)
Comment (oponder) [reply src/components/Users/InvitesTable.js:40]: Looks ok, but kinda weird that we go back and forth on styles. Can we set a global style somewhere? Or instead just take this as our new style?

**Review by oponder — APPROVED** [review]
LGTM in general. 👍

File: src/components/Users/InvitesTable.js (hunk 1 above)
Comment (axbarsan) [reply src/components/Users/InvitesTable.js:40]: This style is just for `react-bootstrap-table-next` tables, and is temporary, until I refactor those, as well (those have to use the more complex `DataTable` `grommet` component)

**Comment by axbarsan** [comment]
Merging so we could use these tables in the RBAC UI

**Pull request merged**